src/main/java/thePackmaster/patches/contentcreatorpack/CharacterTwoAmountPowerStackPatch.java

```diff
@@ -0,0 +1,27 @@
+package thePackmaster.patches.contentcreatorpack;
+
+import basemod.ReflectionHacks;
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import javassist.CtBehavior;
+import thePackmaster.powers.contentcreatorpack.FrostprimePower;
+
+@SpirePatch(clz = ApplyPowerAction.class, method = "update")
+public class CharacterTwoAmountPowerStackPatch {
+    @SpireInsertPatch(locator = Locator.class, localvars = { "p" })
+    public static void stackAmount2(ApplyPowerAction __instance, AbstractPower p) {
+        AbstractPower powerToApply = ReflectionHacks.getPrivate(__instance, ApplyPowerAction.class, "powerToApply");
+        if (p instanceof FrostprimePower && powerToApply instanceof FrostprimePower) {
+            ((FrostprimePower)p).stackAmount2(((FrostprimePower)p).amount2);
```

Comment:
There's a bug with this -- it's double the stacks of the power right now since it pulls the amount to add from `p` instead of from `powerToApply`. 
```suggestion
            ((FrostprimePower)p).stackAmount2(((FrostprimePower)powerToApply).amount2);
```